src/main/java/rpgcharacters/commands/CommandCreateUser.java

```diff
@@ -30,6 +30,19 @@ public void run(Connection conn) {
 
             Statement stmt = conn.createStatement();
             stmt.execute(query);
+
+            query = "GRANT dbUser "
+                  + "TO (SELECT username "
+                  +     "FROM user "
+                  +     "WHERE username = '" + this.username + "');";
+            stmt.executeQuery(query);
+
+            query = "GRANT dbAdmin "
+                  + "TO (SELECT COUNT( * ) AS userCount "
+                  +     "FROM user "
+                  +     "WHERE userCount = 1);";
```

Comment:
I don't think this is valid SQL. We'll probably have to do `SELECT COUNT(*) as userCount FROM user` first. Then do something like
```
if (results.getInt("userCount") == 1) {
    // grant admin to user
} else {
    // grant db user to user
}
```